modules/psap-driver-toolkit-pulling.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/psap-driver-toolkit.adoc
+
+[id="pulling-the-driver-toolkit"]
+= Pulling the Driver Toolkit container image
+
+The driver-toolkit image is available on registry.redhat.io and in the OpenShift release payload. The image corresponding to the most recent minor release of {product-title} will be tagged with the version number on registry.redhat.io. The image URL for a specific release can be found using the `oc adm` CLI command.
```

Comment:
available from the link:https://registry.redhat.io/[Container images section of the Red Hat Ecosystem Catalog]

the version number in the catalog.